apps/sms/views/conversation/js/information.js

```diff
@@ -247,14 +248,17 @@ var VIEWS = {
 
         ConversationView.promptContact({
           number: target.dataset.number
-        });
+        }).then(
+          () => Navigation.toPanel('thread', { id: this.id })
+        );
```

Comment:
see https://dl.boxcloud.com/d/1/6YExEFaKtXJu-PjLownorvuCV7skpd7GXGrcvfXx6X4RQ8KrRgpix4iDtct8ud9RrAcOfX86Q_G2Zmnsr9--BTX8SgrdFhLZUikA2h61s9QFelbzhi_Vuj5i9UldPBpyZf3GX6rcVd8zJFkJTdIMJngZYoEHKwRV0UBVTUR3pXrRUIeiVpQjURGdzAE3ZPtiHlI8g_xUq415PCopLjA0E1m2TyPvk60l_2gPKwuFW8TqYBjhavn48Gghs49Hjfre2m_2OTpuvevMOg5wFfaNNyaxItoJsLaJ7KErsHMGOkqbdtV3VK_GEug2ImFhxUEBLqp9k7rPWk9uGC2fC8XZNSubXKaWIgRABXeXyq7N9S4U-GJHADD0UlD-c6n5yzk9gOLdKwu6DRR773lahnlm6Rhvj6UXc5k2YSsaTCf1GN3vrVr_dI6c4ArdFKcVDn6KeOZYW415dzOQ5sMUdCoAxt1ezwEvOT8RtuUFbLYzm3bwREod2-MDamJHX5uxy-OlVLF-_5H3CthqQ4NqQ4yrH2RpLKVvE4dsCFdy3yeCbb7lu2tCdOpOL9QClgFVDtDYsJ7L0iUmTEZs1PU74yiqEFwXNzV1S7tG1aBPY4Snh_jEFrCHHPpyDdyMVguPmeFUv3FfTiY82Bkmgj0MOBDkSvSJHMV9RfR2CN1h8Sqif0Pjda884N0hh9W_j_p0ZQ2pkmfBj-wzm-0JvAxgSb_mpQl06SK-u0yLM7juA8HYq6Uf6xL5_-LPFtScKOYKRbhGxI_tKUMnUVOkFo8eaW9BBA1Crbl3TOgd9d2X6II1esRsll7ZfSnxzMRsL3N20i5Q03Ol4wgl7PDdkKp490s6ZJSjyDEdRmtQUia3en--B1LUWpwOSBbes9VftgKNVmrSUIquzwX2bbutiLUxBAnMi9szSSJl7ks3pAwQRM6iHeT0SprA12TiEhgxbUS0NymfPjfNAdPGUZfa9dhdfG7QN2qun4mdFrgvJT7fY0vRqyjbVN2roAqeE9uYVqLAvswq4x-5oDiNEUCg4g4UiCji6mhTrTUbonHcI2oj1PFCdrXgC5BZjKrq24dZ6ls_eCmQlslRztyEXgnWrG8MF3bwu1mqSJi5Nh-V-dOgwApnHPyw1qYVY8zBFvnTL4G1H9VZyUwadwu1kAsxDaTY_DM4sDjOxpxvF1n_nmPtiCJJGBZR3tGw2gR1P4H64WDuA49Z3H1jFaHhkvu-VJwuY14dATZbXI4EQoO0dxQZp1WDi3hiBcPjGxm_c4B7NZ0o1y_3KK6Zc40XjtDQY33skWwHcJlKCD5z9qAceTOmQr6o9ie7jD66cqkdfC46kW3pGUkCj1zbTFkEfNSRKwtuROlg/download page 15 :)

(sorry for the very big URL, other URLs don't work properly... I should just upload old spec on the wiki eventually)